### ISSUE TITLE
1.5.2 - Download content directly in various formats (renamed, now 3.2.7, Content download) 

### ISSUE THREAD

**Issue by janahoffmann** [gbhl/bhl-europe#71]
Download item directly in various formats (PDF, OCR, TIFF, etc.), not from content viewer but from ‘bibliographic information page’ and 'result list' (only for articles and monographs, maybe volumes). 
Start download by clicking on icon for format; or redirect to content viewer (optional: add content to batch download basket)
Display size of to be downloaded items. 

Precondition: 
File size in metadata or algorithm for estimation of file size.

**Comment by sprogerb** [comment]
Relates to Tervuren-Protocol (issue 12) -  AIT contributions scheduled within 22.Aug. - 31.Aug.

**Comment by sprogerb** [comment]
POrtal Issue

**Comment by lobajuluwa** [comment]
Update from Portal Review Workshop: already fixed for development version - done.

Jiri - I've moved this into the next Milestone Release for November. does it still need testing?
@JiriFrank

**Comment by JiriFrank** [comment]
@lobajuluwa
Yes, it needs to be tested before release, because we cannot check it right now without the content and functional result list. It was demonstrated (In The Hague) on the content viewer but not on the result list (portal basket).

**Comment by lobajuluwa** [comment]
@JiriFrank 
@chris-sleep
tagging for direct download.
Issue: review content viewer download - broken link?

**Comment by JiriFrank** [comment]
The tagging basket is set up and also the download of various metadata from the biblio page. 

Download of jpg, ocr and pdf is ready but not active yet. We have just one sample at the moment. http://bhl-int.nhm.ac.uk/portal/bhle-view/bhle%3A10706-a0tqjq56?language=fi

The source will open in new tab.

But we can test it. More active links will come later.
